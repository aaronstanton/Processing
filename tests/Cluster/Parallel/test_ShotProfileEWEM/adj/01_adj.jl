using Seismic

SCRATCH_PATH = "/global/scratch/stanton/kevins_model/elastic/"

# initialize only 1 process per node
#Seismic.bind_torque_procs(1)
#@everywhere using Seismic

# for each node, migrate 1 common shot gather using 12 threads per node. 
param = {"adj"=>true,
    "vp"=>join([SCRATCH_PATH,"prep/vp"]), "vs"=>join([SCRATCH_PATH,"prep/vs"]), "wav"=>join([SCRATCH_PATH,"prep/wav"]),
    "angx"=>join([SCRATCH_PATH,"prep/angx"]),"angy"=>join([SCRATCH_PATH,"prep/angy"]),
    "pade_flag"=>"n",
    "damping"=>10000000,
    "sz"=>10, "gz"=>550,
    "fmin"=>1., "fmax"=>50.,
    "padt"=>2, "padx"=>2,
    "nangx"=>29, "oangx"=>-70, "dangx"=>5,
    "nangy"=>1, "oangy"=>0, "dangy"=>2,
    "nhx"=>827, "ohx"=>-3304, "dhx"=>8,
    "nhy"=>1, "ohy"=>0, "dhy"=>8,
    "verbose"=>true};
#param["sx"] = [100.0:100.0:6900.0];
#param["sy"] = [100.0:100.0:6900.0]*0;
param["sx"] = [3600.0:100.0:3600.0];
param["sy"] = [3600.0:100.0:3600.0]*0;
m = [join([SCRATCH_PATH,"adj/mpp"]);join([SCRATCH_PATH,"adj/mps1"]);join([SCRATCH_PATH,"adj/mps2"])]
d = [join([SCRATCH_PATH,"prep/ux"]);join([SCRATCH_PATH,"prep/uy"]);join([SCRATCH_PATH,"prep/uz"])]
ShotProfileEWEM(m,d,param);

