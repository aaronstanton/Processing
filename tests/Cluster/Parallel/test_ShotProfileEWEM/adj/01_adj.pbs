#!/bin/bash 
#PBS -S /bin/bash
#PBS -l nodes=3:ppn=12 
#PBS -l mem=66gb 
#PBS -l walltime=1:00:00 
#PBS -m bea
#PBS -M aaron.stanton@ualberta.ca
#PBS -N 01_adj
#PBS -o 01_adj.out
#PBS -e 01_adj.err

date

# Change to directory from which job was submitted
cd $PBS_O_WORKDIR 

echo "here is your PBS_NODEFILE"
cat $PBS_NODEFILE | uniq > machines.txt
cat machines.txt

export OMP_NUM_THREADS=12

echo "calling julia now"
julia --machinefile machines.txt tmp.jl

