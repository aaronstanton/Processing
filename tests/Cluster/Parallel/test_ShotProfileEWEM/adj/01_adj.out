Fri Nov 20 20:54:06 MST 2015
here is your PBS_NODEFILE
cn0683
cn0683
cn0683
cn0683
cn0683
cn0683
cn0683
cn0683
cn0683
cn0683
cn0683
cn0683
cn0735
cn0735
cn0735
cn0735
cn0735
cn0735
cn0735
cn0735
cn0735
cn0735
cn0735
cn0735
cn0738
cn0738
cn0738
cn0738
cn0738
cn0738
cn0738
cn0738
cn0738
cn0738
cn0738
cn0738
calling julia now
